src/app/modules/main/communities/module.nim

```diff
@@ -110,7 +110,20 @@ method getCommunityItem(self: Module, c: CommunityDto): SectionItem =
           onlineStatus = OnlineStatus.Offline, # TODO get the actual status?
           isContact = contactDetails.details.added, # FIXME
           )),
-      historyArchiveSupportEnabled = c.settings.historyArchiveSupportEnabled
+      historyArchiveSupportEnabled = c.settings.historyArchiveSupportEnabled,
+      bannedMembers = c.bannedMembersIds.map(proc(bannedMemberId: string): MemberItem =
+        let contactDetails = self.controller.getContactDetails(bannedMemberId)
+        result = initMemberItem(
+          pubKey = bannedMemberId,
+          displayName = contactDetails.displayName,
+          ensName = contactDetails.details.name,
+          localNickname = contactDetails.details.localNickname,
+          alias = contactDetails.details.alias,
+          icon = contactDetails.icon,
+          onlineStatus = OnlineStatus.Offline, # TODO get the actual status?
+          isContact = contactDetails.details.added, # FIXME
```

Comment:
What is this `FIXME` comment about? Can we get at least a bit more context on **what** needs to be fixed and why?

Reply:
This is a question to @osmaczko. Patryk, this FIXME is in a few places, maybe we can create a ticket to fix this ?

Reply:
I believe it was already fixed by @jrainville -> https://github.com/status-im/status-desktop/pull/6375/